apps/service_providers/llm_service/runnables.py

```diff
@@ -405,39 +408,27 @@ def _save_response_annotations(self, output, thread_id, run_id) -> tuple[str, di
             resource, _created = chat.attachments.get_or_create(tool_type="image_file")
             resource.files.add(*image_file_attachments)
 
-        return output_message, list(file_ids)
+        return output_message.strip(), list(file_ids)
```

Comment:
Should this be `output_message.strip('\n')`, since we might be adding an extra newline character?

Reply:
nevermind, I see strip removed newlines as well by default